### PR TITLE
adding gif

### PR DESCRIPTION
**Pull request by RoshniNaveenaS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ cds compile <path to srv folder> --to ord [-o] [destinationFilePath]
 1) Run `cds watch` in the application's root.
 2) Check the following relative paths for ORD information - `/.well-known/open-resource-discovery` , `/open-resource-discovery/v1/documents/1`.
 
+![](ord.gif)
```
Comment (jeevitha011) [inline README.md:41]: you should use image tag like [this](https://github.com/cap-js/notifications/blob/main/README.md?plain=1#L114)

File: README.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ cds compile <path to srv folder> --to ord [-o] [destinationFilePath]
...
+
```
Comment (jeevitha011) [inline README.md:42]: the ord endpoint url is not visible in the gif

Comment (jeevitha011) [inline README.md:42]: - file size is too big and should be compressed as it takes time to load
- gif should be sped up as it is very slow

Comment (jeevitha011) [inline README.md:42]: 'site cant be reached' part of the gif should be edited out

File: README.md [hunk 3]
```diff
@@ -39,6 +39,9 @@ cds compile <path to srv folder> --to ord [-o] [destinationFilePath]
 2) Check the following relative paths for ORD information - `/.well-known/open-resource-discovery` , `/open-resource-discovery/v1/documents/1`.
 
 
+<img width="1300" alt="Sample Application Demo" style="border-radius:0.5rem;" src="ord.gif">
```
Comment (jeevitha011) [inline README.md:42]: ord.gif should be in a separate folder called asset/etc

**Comment by RoshniNaveenaS** [comment]
Have added the updated gif

File: README.md (hunk 3 above)
Comment (RoshniNaveenaS) [reply README.md:42]: made the changes

File: README.md (hunk 2 above)
Comment (RoshniNaveenaS) [reply README.md:43]: now it is

Comment (RoshniNaveenaS) [reply README.md:43]: have increased the speed

Comment (RoshniNaveenaS) [reply README.md:43]: have removed

File: README.md (hunk 3 above)
Comment (muskansethi1) [inline README.md:42]: update src, the gif is not loading in the readme

File: README.md [hunk 4]
```diff
@@ -39,6 +39,9 @@ cds compile <path to srv folder> --to ord [-o] [destinationFilePath]
...
+
+
```
Comment (muskansethi1) [inline README.md:44]: For me, the gif goes to white and gets stuck on it after one point, is it supposed to be that way?

File: README.md (hunk 3 above)
Comment (RoshniNaveenaS) [reply README.md]: made changes

File: README.md (hunk 4 above)
Comment (RoshniNaveenaS) [reply README.md:44]: have updated that can u check again

**Comment by muskansethi1** [comment]
personal opinion - we could use an incognito browser window, so that the gif doesn't have the bookmarks bar or any other info. It would look cleaner, and avoid any personal info being displayed

**Comment by RoshniNaveenaS** [comment]
> imo - we could use an incognito browser window, so that the gif doesn't have the bookmarks bar or any other info. It would look cleaner, and avoid any personal info being displayed

made changew

**Review by muskansethi1 — APPROVED** [review]

**Pull request merged**